packages/govuk-frontend/src/govuk/template.jsdom.test.js

```diff
@@ -2,8 +2,16 @@ const crypto = require('crypto')
 
 require('html-validate/jest')
 
-const { renderTemplate } = require('@govuk-frontend/helpers/nunjucks')
-const { nunjucksEnv } = require('@govuk-frontend/lib/components')
+const {
+  nunjucksEnv,
+  renderTemplate
+} = require('@govuk-frontend/lib/components')
+
+function replacePageWith(contents) {
+  document.open()
+  document.write(contents)
+  document.close()
```

Comment:
Nice one for replacing the whole document content 🙌🏻

Reply:
The tests didn't work otherwise as anything that was meant to be in the `<head>` ended up in the `<body>`!